python/py-healpy/Portfile

```diff
@@ -31,8 +31,7 @@ if {${name} ne ${subport}} {
 
     variant openmp description "enable OpenMP parallel acceleration" {
         # Pick a compiler that supports openmp
-        compiler.whitelist macports-clang-6.0 macports-clang-5.0 macports-clang-3.7 macports-gcc-7 macports-gcc-6 macports-gcc-5 macports-gcc-4.8 macports-gcc-4.7 macports-gcc-4.6 macports-gcc-4.5 macports-gcc-4.4 macports-gcc-4.3
-        compiler.fallback macports-clang-6.0
+        compiler.openmp_version 2.5
```

Comment:
Is this version of OpenMP appropriate? I did not determine from briefly looking at the upstream project if a higher version is required, so I've used the minimum possible (2.5).

Also, is this change fine without a revision bump?

Reply:
> Is this version of OpenMP appropriate? I did not determine from briefly looking at the upstream project if a higher version is required, so I've used the minimum possible (2.5).

Yes, I think that should be fine.

Reply:
> Also, is this change fine without a revision bump?

Yes, I think so.